src/lib/buildWebpackEntries.js

```diff
@@ -55,33 +84,31 @@ export function getWebpackEntries () {
  * includes the generated entry point. Finally, if we are in development mode
  * it will start the array off with the webpack hot middleware client.
  */
-export default function buildWebpackEntries (isProduction) {
+export default function buildWebpackEntries (isProduction, entrypointToBuild) {
   const output = {};
   const basePath = getBasePath();
 
   // Clean slate
   fs.removeSync(basePath);
   fs.ensureDirSync(basePath);
 
-  const entries = getWebpackEntries();
-  for (const key in entries) {
-    const entry = entries[key];
-    const { filePath, fileName } = entry;
-    fs.outputFileSync(filePath, getEntryPointContent(entry));
-    output[fileName] = [path.join(__dirname, "..", "entrypoints", "client.js"), filePath];
-
-    // Include hot middleware in development mode only
-    if (!isProduction) {
-      output[fileName].unshift("webpack-hot-middleware/client");
-
-      // eventsource-polyfill added here because it needs to go before
-      // webpack-hot-middleware pointing to local node_modules so it pulls from
-      // gluestick project and not from the app. This is needed to support HMR in
-      // browsers like IE11
-      output[fileName].unshift(path.join(__dirname, "..", "..", "node_modules", "eventsource-polyfill"));
+  const { mapEntryToGroup, entries } = getWebpackEntries();
+  if (entrypointToBuild && entrypointToBuild[0] === "/") {
+    const entry = parseWebpackEntry(entries[entrypointToBuild], isProduction);
+    output[entry.fileName] = entry.conent;
+    return output;
+  } else if (entrypointToBuild) {
+    const filtredEntries = filterEntries(entries, mapEntryToGroup[entrypointToBuild]);
+    for (const key in filtredEntries) {
```

Comment:
this for in loop should be moved to separate function because later (line 108) there's the same for in loop, so to keep things DRY those two loops should be replaced with function call and this function will have this for in loop